angular.json

```diff
@@ -89,6 +91,8 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/bootstrap/dist/css/bootstrap.min.css",
```

Comment:
@Yuriy-27 you don't need it here, it's for testing purposes but you're not gonna use it for now, and I think at all